Dockerfile

```diff
@@ -7,6 +7,8 @@ RUN apk add --update --no-cache \
   libxslt-dev \
   # needed for mysql2 dependency
   mariadb-dev \
+  # needed because mimemagic-a-palooza
+  shared-mime-info \
```

Comment:
Is this still needed since mimemagic has been removed?

Reply:
apparently not. removing. thanks!